plugin/code_lens.py

```diff
@@ -82,6 +83,7 @@ def _region_key(self, index: int) -> str:
         return '{0}.{1}'.format(self.CODE_LENS_KEY, index)
 
     def clear_view(self) -> None:
+        self._phantom.update([])
         self.clear_annotations()
```

Comment:
Maybe rename the method now since it can't be either annotation or phantom? Something more generic that doesn't mention the UI concept. Maybe just `clear()` since the class name `CodeLensView` already makes it clear that it's something UI related.

Reply:
No, sorry, this is actually only about annotations.